mcafee_atd/plugin.spec.yaml

```diff
@@ -643,3 +643,53 @@ actions:
         description: Returns true if found information about analyzer profiles
         type: boolean
         required: true
+  get_report:
+    title: Get Report
+    description: Download the analysis report file for given parameter
+    input:
+      id:
+        title: ID
+        description: Task ID or job ID or MD5 value for which will be prepared analysis report
+        type: string
+        required: true
+        example: 13
+      type_id:
+        title: Type ID
+        description: Type of given ID parameter, default value is MD5
+        type: string
+        required: false
+        default: MD5
+        example: TASK ID
+        enum:
+          - MD5
+          - TASK ID
+          - JOB ID
+      report_type:
+        title: Report Type
+        description: Type of file prepared analysis report
+        type: string
+        required: false
+        default: JSON
+        example: HTML
+        enum:
+          - JSON
+          - HTML
+          - TXT
+          - ZIP
+          - XML
+          - IOC
+          - STIX
+          - PDF
+          - SAMPLE
+    output:
+      file:
+        title: File
+        description: Prepared analysis report
+        type: bytes
+        required: true
+      report:
+        title: Report
+        description: Return report in JSON
+        type: object
+        required: false
```

Comment:
We should always return the JSON output and the file output - for any selected file type.

Reply:
So we should remove `report_type` and returned always only JSON from API and in this case we never returned PDF, HTML or other types?

Reply:
@r7-kszczepanskagorna Ah, I was thinking we make two calls in this action. 

1. One call to always get the JSON report and return it in `report_type`
2. One call to get the file the user selected and return it in `file`

A user may want to make a decision with the JSON in the workflow and then later attach a file to a ticket for example without adding two actions to the workflow.

Reply:
Ok. So we will add second call always for JSON report and we remove json from enum type.